source/chapters/library.rst

```diff
@@ -866,6 +867,77 @@ different location. Select :guilabel:`Use Default Library` to reset.
+
+Your crates in Serato DJ Pro can also be imported into Mixxx. This feature exhibits Mixxx customization, as your favourite music collection are imported and arranged as they were.
+
+You can also import your Serato library into Mixxx that are in USB drives for a portable Serato library.
+
+**Set-up**
+
+Copy music library from Serato into external USB drive
+Plug in USB drive while Mixxx is running
+Click on Serato in the library, all records on USB from Serato will be visible (attach as many as you like!)
+Then click on the desired removable device, and all your folders and playlists from Serato will be revealed for you to play from
+
+Import your Beatgrid and Hotcues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Track cue points are saved in special file tags as well as the track metadata. 
+You can now import your track cue points and all of it's metadata from Serato to Mixxx.
+
+Once you import your Serato library into Mixxx and load a track for the first time from any of the methods above, the positions, labels and colors of your hotcues, other metadatas and beatgrid are automatically imported.
+
+If you have already added a track to your Mixxx library before support for reading Serato's hotcues was added, you can reimport metadata from Serato via the track context menu.
+This action will clear your existing cuepoints in Mixxx if the track has any Serato hotcues.
+
+Decodes may slightly detect different track start and end times for files from different sources, this could possibly be because of the presence of countless encoders and decoders for MP3 and M4A/AAC.
+This problem may cause your cues to be shifted up by a few milliseconds. 
+While we are working very hard to mitigate this problem, we have added a way to shift all cues for a track at once. 
+With this method, it possible to fix the cue positions if Mixxx fails to determine the correct offsets.
+
+Below are lists of tags that are currently read and parsed by Mixxx
+
+
+**Set-up**
+
+Import Loops from Serato
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+From Mixxx 2.3, saved loops from Serato can be imported. Proper support for importing and restoring saved loops from Serato is scheduled for Mixxx 2.4. 
+
+For now, we just import these loops into the Mixxx database and allows using their start position as regular hotcues. 
+With this in place, you can start using your loops once Mixxx 2.4 is out and avoid reimporting track meta data.
+
+**Set-up**
+
+Imports not currently supported
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Support for importing track overview waveform image stored in Serato into Mixxx has not been provided in Mixxx 2.3. 
+Serato flip macros cannot be imported either because flip does not offer that kind of functionality yet.
+
+Currently, you cannot write cues from Mixxx into Serato metadata tags. 
+We are working hard to implemment support for exporting Serato metadata tags in upcoming Mixxx versions, stay tunned.
+
```

Comment:
Thank you for writing this. However, the phrasing is a bit "marketing-like" and for the manual we can make this a lot shorter and more formal.

I think we just need to describe the following things briefly:
- It's possible to read Serato libraries from your hard disk and USB drives, including crates. Smart crates are not supported at the moment.
- Mixxx will also read Serato metadata from file tags and will import the beatgrid and hotcues from the tags. Saved Loops will be imported too, but as of Mixxx 2.3 only the first loop is usable.
- If you want to reimport the metadata for files, you can right-click on a track and select Metadata > Import from file tag. If the track has Serato Metadata, all existing cue points will be overwritten.
- Exporting the beatgrid and hotcues to Serato is currently not supported.

Reply:
ok! I would look into this now!

Reply:
> Thank you for writing this. However, the phrasing is a bit "marketing-like" and for the manual we can make this a lot shorter and more formal.
> 
> I think we just need to describe the following things briefly:
> 
> * It's possible to read Serato libraries from your hard disk and USB drives, including crates. Smart crates are not supported at the moment.
> * Mixxx will also read Serato metadata from file tags and will import the beatgrid and hotcues from the tags. Saved Loops will be imported too, but as of Mixxx 2.3 only the first loop is usable.
> * If you want to reimport the metadata for files, you can right-click on a track and select Metadata > Import from file tag. If the track has Serato Metadata, all existing cue points will be overwritten.
> * Exporting the beatgrid and hotcues to Serato is currently not supported.

I just worked on this, thank you for your time and the **honest feedback you gave**. Changes have been done to effect this, please let me know if there's anything to add or remove to make this documentation better.

Don't you think we need an image for this section? Any suggestions?